README.md

```diff
@@ -1,51 +1,39 @@
 # LimeLight
-> A disaster management ecosystem from the future
+> Disaster relief ecosystem to locate victims and coordinate operations
```

Comment:
IMO subheading can be kept short to just `Disaster relief ecosystem` or `Disaster management ecosystem` just to retain people's interest till the end of README. We're anyway describing what the app does in the following paragraph.

Reply:
I'm not sure whether `disaster relief ecosystem` or `disaster management system` gives enough context. What does one make of it? It sounds generic to me.

Reply:
But yes, on the other hand, sub-heading can be kept short as well so I'm really not sure.

Reply:
@kritisingh1 can you please make the change on this branch removing the part after ecosystem?